core/src/main/scala/scalaz/Order.scala

```diff
@@ -85,20 +85,32 @@ object Order {
 
   ////
 
-  implicit val orderInstance: Divisible[Order] = new Divisible[Order] {
-    override def contramap[A, B](r: Order[A])(f: B => A) = r.contramap(f)
```

Comment:
it looks like you dropped `contramap`?

Reply:
Yes sorry, that was because I dropped Divisible[Order] instance in first impl. Adding again